metasyn/config.py

```diff
@@ -0,0 +1,171 @@
+"""Configuration classes for creating metaframes."""
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Iterable, Optional, Union
+
+try:
+    import tomllib
+except ImportError:
+    import tomli as tomllib  # type: ignore  # noqa
+
+from metasyn.privacy import BasePrivacy, get_privacy
+from metasyn.provider import DistributionProviderList
+from metasyn.util import VarConfig
+
+
+class MetaConfig():
+    """Configuration class for creating MetaFrames.
```

Comment:
```suggestion
    """
    Manages configurations for individual variables in a MetaFrame.

    This class is used to create, manipulate, and retrieve configurations for
    individual variables in a MetaFrame. It also provides methods for loading
    configurations from .toml files and converting configurations to dictionaries.
```

Feel free to change it to your liking, but I think it's nice to have it be a bit more descriptive from the start.